code/modules/mob/living/silicon/robot/robot_modules/station_vr.dm

```diff
@@ -179,6 +182,7 @@
 	//R.icon_state = "scrubpup"
 	R.pixel_x 	 = -16
 	R.old_x 	 = -16
+	R.dogborg = 1
```

Comment:
Current standard is `TRUE` and `FALSE` for boolean stuff. Just make these `TRUE` and the other one `FALSE`